quiz/src/Category.js

```diff
@@ -0,0 +1,43 @@
+import React ,{useState,useEffect} from 'react';
+import {CategoryData,DifficultyData} from './Data';
+
+function Category () {
+  const[cat,setCat]=useState('');
+  const[diff,setDiff]=useState('');
+  const [questions,setQuestions]=useState([]);
+  const[show,setShow]=useState(true);
+  useEffect(()=>{
+
+    if(cat||diff){
```

Comment:
good work here at validating values, that will prevent purposeless requests :+1: